web/src/components/tags-input/index.tsx

```diff
@@ -143,28 +133,9 @@ export const TagsInput: React.FC<TagsInputProps> = props => {
         ref={ref}
       >
         <div className={classes.scrollbar}>
-          {tags.map(tag => {
-            const { id, firstName, avatar, lastName, displayName } = tag
-            const name =
-              !firstName || !lastName ? displayName : `${firstName} ${lastName}`
-
-            return (
-              <Tag
-                key={id}
-                className={classes.tag}
-                avatarSrc={avatar}
-                avatarAlt={name}
-                onRemove={() => {
-                  onRemove(id)
-                }}
-              >
-                {name}
-              </Tag>
-            )
-          })}
           {children &&
             children.map(child => (
-              <div key={genUid()} className={classes.tag}>
+              <div key={child.props.id} className={classes.tag}>
```

Comment:
why do we manually need to set an id on the `Tag`?

Reply:
Good point, thanks